api/src/main/java/io/grpc/Server.java

```diff
@@ -104,8 +104,8 @@ public int getPort() {
 
   /**
    * Initiates an orderly shutdown in which preexisting calls continue but new calls are rejected.
-   * After this call returns, this server has released the listening socket(s) and may be reused by
-   * another server.
+   * This call will not wait for preexisting calls finish before returning. After this call returns,
+   * this server has released the listening socket(s) and may be reused by another server.
```

Comment:
I suggest the following:

>  Initiates an orderly shutdown to allow preexisting calls to continue but new calls are
rejected. {@link #awaitTermination()} or {@link #awaitTermination(long, TimeUnit)} needs 
to be called to wait for existing calls to finish. After this call returns, this server
has released the listening socket(s) and may be reused by another server.

Reply:
Combined your and @ejona86's suggestions: https://github.com/grpc/grpc-java/pull/6512/commits/af9a157931771751dacddc334f578c557e6afaac